### DISCUSSION TITLE
"Master" in naming convention

### DISCUSSION THREAD

**Discussion by abachma2** [cyclus/cyclus#1704] [General]
We've updated the default branches of Cyclus, Cycamore, and Cymetric to use 'main' instead of 'master'. But we have "master schema" as part of the language in the Organization. Is this something we want to update? If so, how? Potential new names could be "primary schema" or "main schema". 

Are there any other instances where "master" may be in the ecosystem language?

**Comment by nsryan2** [comment]
I would be down to look in to other uses of `master`. I think main schema works as a substitute for `master schema`.

Some other uses in the cyclus org:
Cyclus.github.io
----------------
* `master_doc`: https://github.com/cyclus/cyclus.github.com/blob/6f011bbe675235b1a13fc344550dbac381db5525/source/conf.py#L76

Remotecyclus
--------------
* `SimulationServermaster` https://github.com/cyclus/remotecyclus/blob/e3844d3aa736de62c990e8e2bcf79143ec7a4b3b/uidaho/remote/SimulationServermaster.java#L6

Cymetric
---------
* `master_ci` https://github.com/cyclus/cymetric/blob/f44337883500d8e0875dbc00900df556a5c00c7b/docker/master-ci/Dockerfile#L3

Cycamore
----------
* `master` branch https://github.com/cyclus/cycamore/blob/129cce7a2bd3f907c20fde26f4acf8084a62873a/CHANGELOG.rst#L21

Rickshaw
---------
* `master_doc` https://github.com/cyclus/rickshaw/blob/108a546fe6f1bb3a94e2443e05eed1c9f8d600e3/docs/conf.py#L47

Recycle
--------
* `master branch` https://github.com/cyclus/recycle/blob/04723465b088ce779a77d221526dfede8ef1e7c7/README.rst#L33
